test/transforms.jl

```diff
@@ -328,6 +328,50 @@
     @test n1 == n2
   end
 
+  @testset "Filter" begin
+    # Test with no_missing_in_row
+    x1 = [1, 2, 3, 4, 5]
+    x2 = [missing, 2, 3, 4, 5]
+    x3 = [5, 5, 5, 5, 5]
+    x4 = [1, 2, 3, 4, 5]
+    x5 = [1, 2, missing, 4, 5]
+    table = Tables.table([x1 x2 x3 x4 x5])
```

Comment:
Please use `t = Table(; x1, x2, x3, x4, x5)` instead like in other tests. This is a table type from TypedTables.jl.